.vib/owncloud/cypress/cypress/integration/owncloud_spec.js

```diff
@@ -4,77 +4,36 @@ import { random } from '../support/utils';
+  cy.contains('Folder').click();
   cy.fixture('folders').then((folder) => {
     cy.get('[value="New folder"]').type(`${folder.newFolder.name}.${random}`);
     cy.contains('Create').click();
-    cy.contains('.innernametext', `${folder.newFolder.name}.${random}`).click();
-  });
-  cy.get('.new').click();
-  cy.get('[data-action="upload"]').click();
-  cy.get('input[type="file"]').selectFile(
-    'cypress/fixtures/file_to_upload.json',
-    { force: true }
-  );
-  cy.contains('No files in here').should('not.be.visible');
-});
-
-it('allows adding a group and a user', () => {
-  cy.login();
-  cy.visit('/index.php/settings/users');
-  cy.contains('Add Group').click();
-  cy.fixture('groups').then((group) => {
-    cy.get('#newgroupname').type(`${group.newGroup.name}.${random}`);
-    cy.get('#newgroup-form').within(() => {
-      cy.get('[type="submit"]').click();
+    cy.get('span').contains(`${folder.newFolder.name}.${random}`).click();
+    cy.contains('No files in here');
+    cy.get('.new').click();
+    cy.get('[data-action="upload"]').click();
+    cy.get('[type="file"]').selectFile(
+      'cypress/fixtures/file_to_upload.json',
+      { force: true }
+    );
+    cy.contains('file_to_upload').should('be.visible');
+    cy.get('[href*="file_to_upload"]').within(() => {
+      cy.get('[data-action="Share"]').click();
     });
-    cy.contains('.groupname', `${group.newGroup.name}.${random}`);
-  });
-  cy.fixture('users').then((user) => {
-    cy.get('#newusername').type(`${user.newUser.name}.${random}`);
-    cy.get('#newemail').type(`${user.newUser.email}.${random}`);
-    cy.contains('Create').click();
-    cy.contains('.name', `${user.newUser.name}.${random}`);
-  });
-});
-
-it('checks the SMTP configuration', () => {
-  cy.login();
-  cy.visit('/index.php/settings/admin?sectionid=general');
-  cy.get('#mail_smtphost').should('have.value', Cypress.env('smtpHost'));
-  cy.get('#mail_smtpname').should('have.value', Cypress.env('smtpUser'));
-  cy.get('#mail_to_address').should('have.value', Cypress.env('owncloudEmail'));
-});
-
-it('allows sharing a file by link', () => {
-  cy.login();
-  cy.get('[data-file="ownCloud Manual.pdf"]').within(() => {
-    cy.get('[data-action="Share"]').click();
-  });
-  cy.get('[class="permalink"]').click();
-  cy.get('.detailFileInfoContainer').within(() => {
-    cy.get('input')
+    // Create a public link and try to download the file
+    cy.contains('Public Links').click();
+    cy.contains('Create public link').click();
+    cy.get('button').contains('Share').click();
+    cy.get('.linkText')
       .invoke('val')
       .should('contain', 'vmware')
       .then((link) => {
         cy.request({
-          method: 'GET',
-          url: link,
-          form: true,
+          url: `${link}/download`,
         }).then((response) => {
-          expect(response.status).to.eq(200);
-        });
+          const mime = response.body
+          expect(response.body).to.contain('testJson')
```

Comment:
This test will fail if the content of the example file is changed. Instead, it is a better practice to read the content of the file and compare it to the downloaded one.

Please, take a look to 

https://github.com/bitnami/charts/blob/6d9b5a9ad6007c2e27a936d3f800b00425aae15d/.vib/minio/cypress/cypress/integration/minio_spec.js#L26 

for a similar test